.claude/skills/mastra-smoke-test/scripts/test-server.sh

```
@@ -0,0 +1,77 @@
+echo "Calling $AGENT_ID with: \"$MESSAGE\""
+echo ""
+
+# Use jq for safe JSON construction (handles special characters in MESSAGE)
+JSON_BODY=$(jq -n --arg msg "$MESSAGE" '{"messages":[{"role":"user","content":$msg}]}')
+RESPONSE=$(curl -sS --connect-timeout 10 --max-time 60 -w "\n---HTTP_STATUS:%{http_code}---" \
+    -X POST "$SERVER_URL/api/agents/$AGENT_ID/generate" \
+    -H "Content-Type: application/json" \
+    -d "$JSON_BODY")
+
+# Parse response and status
+BODY=$(echo "$RESPONSE" | sed 's/---HTTP_STATUS:[0-9]*---$//')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)